config/initializers/0_print_time_spent.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# This class is taken from https://github.com/nevir/Bumbler which
+# depends on bundler. But for RPM environment, Foreman does not
+# use bundler (but bundler_ext stub). The gem won't load in that
+# environment.
+module ForemanInitializers
+  @slow_threshold = 100.0
+  @slow_items = {}
+
+  class << self
+    attr_writer :slow_threshold
+
+    attr_reader :slow_items
+
+    def benchmark(key)
+      start = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+      result = yield
+      time = (Process.clock_gettime(Process::CLOCK_MONOTONIC) - start) * 1000 # ms
+      @slow_items[key] = time if time > @slow_threshold
+      [time, result]
```

Comment:
it looks like you only use `result`, not `time`. Should it be simplified to
```suggestion
      result
```

Reply:
Ah yeah another leftover from my rip-off.